amundsen_application/static/package.json

```diff
@@ -39,11 +53,10 @@
     "@types/react-router": "^4.4.5",
     "@types/react-tagsinput": "^3.19.7",
     "@types/webpack": "^4.41.12",
-    "babel-jest": "^24.9.0",
+    "babel-jest": "^23.4.2",
```

Comment:
Surprising to see a downgrade. Was this done automatically when making other changes or was there some issue with the previous version where you chose to intentionally downgrade?

Could you also provide context on the babel changes, did it all just come from the move to `core-js`? We've sadly only touched this stuff on as as-needed basis since the first working configuration we had and want to make sure I understand for future maintenance.

Reply:
Oh, good catch!
Yes, this was the babel upgrade script, I will retrieve the previous one, it will probably work the same.

Yes, happy to extend on the babel updates. On version 7.4 of babel they deprecated @babel/polyfill, that it was using core-js internally. Now, instead of using the polyfills, we are going to use that core-js library directly. 

One of the main changes is the move from using the yearly presets and "stage-0" setup into the "preset-env", which uses a list of browsers instead. This list is on the package.json file. This change means that we need to bring the required plugins one by one. 

That said, if we are strict about only supporting Chrome, we could probably get rid of most of this plugins and save some space. Food for thought!

Reply:
Alright sounds good. We're not strict about only supporting Chrome. It's just the only thing currently promise our users because of the nature of not being able to guarantee resources to fix cross-functionality issues.